README_zh-CN.md

```diff
@@ -47,6 +47,8 @@
 
 **新闻**：
 
+**默认分支已经从 `master` 切换到 `1.x`。我们鼓励用户迁移到最新版本，请参考 [迁移指南](https://mmdetection3d.readthedocs.io/zh_CN/dev-1.x/migration.html) 以了解更多细节。**
+
```

Comment:
The type of link for migration guide doc is not unified in English and Chinese doc